lib/elixir/lib/map_set.ex

```diff
@@ -38,7 +38,12 @@ defmodule MapSet do
   """
   @spec new(Enum.t) :: t
   def new(enumerable) do
-    Enum.reduce(enumerable, %MapSet{}, &put(&2, &1))
+    map =
```

Comment:
The same as https://github.com/elixir-lang/elixir/pull/4415/files/1a3ef8fff5413aca7c39658c289d5232440bd235#r56796580.